script/push-l10n.py

```diff
@@ -60,6 +60,8 @@ def main():
     upload_source_strings_desc(source_string_path, filename)
     if 'ethereum-remote-client' in source_string_path:
         upload_missing_json_translations_to_transifex(source_string_path)
+    if 'brave-site-specific-scripts' in source_string_path:
+        upload_missing_json_translations_to_transifex(source_string_path)
```

Comment:
would it make sense to use compound logic to avoid copy/pasting the function call? i.e
```suggestion
    if ('ethereum-remote-client' in source_string_path or
        'brave-site-specific-scripts' in source_string_path):
        upload_missing_json_translations_to_transifex(source_string_path)
```

Reply:
Yes, fixed. Thanks!